kie-remote/pom.xml

```diff
@@ -16,6 +16,10 @@
           <groupId>org.apache.kafka</groupId>
           <artifactId>kafka-clients</artifactId>
       </dependency>
+      <dependency>
+          <groupId>org.kie</groupId>
+          <artifactId>kie-api</artifactId>
```

Comment:
I understand the reason why you are doing it but I don't like because this brings all kie-api methods to the classpath so user can easily initialize a kieSession instead of a remoteKieSession.
I would like really to remove it, it is too error prone from my point of view.
Personally I would prefer a check based on class canonical name instead of add this dependency.
Wdyt?

Reply:
I moved this timestamp checks to hacep as you suggested.